order/src/main/java/com/yas/order/model/Order.java

```diff
@@ -22,16 +20,21 @@ public class Order extends AbstractAuditEntity{
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String phone;
-    private String address;
+    private String email;
+    private Long shippingAddressId;
+    private Long billingAddressId;
```

Comment:
Following my understanding, these fields will map to the user address by id.
There is a scenario that an order that was finished in the past with an example address as "Tan Binh district, Etown 4". Unfortunately, user edits this address to "Thu Duc City, Giga Mall" then the shipping address or billing address of this order will change accordingly. 
It doesn't make sense for shipping or billing address of the past order can be changed.

Reply:
Thank you, i will change it later in the next PR